<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="txtID">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ID Hoạt động</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTen">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tên hoạt động</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDiaDiem">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="placeholderText">
    <string>Địa điểm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quản lí hoạt động</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xác nhận</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngày tổ chức:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtNoiDung">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Nội dung</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
